Run/SetupConfiguration.ps1

```diff
@@ -34,6 +34,10 @@ if ($developerServicesKeyExists) {
     $customConfig.SelectSingleNode("//appSettings/add[@key='DeveloperServicesEnabled']").Value = "true"
     $CustomConfig.SelectSingleNode("//appSettings/add[@key='DeveloperServicesSSLEnabled']").Value = $servicesUseSSL.ToString().ToLower()
 }
+$enableSymbolLoadingAtServerStartupKeyExists = ($customConfig.SelectSingleNode("//appSettings/add[@key='EnableSymbolLoadingAtServerStartup']") -ne $null)
+if ($enableSymbolLoadingAtServerStartupKeyExists) {
+    $customConfig.SelectSingleNode("//appSettings/add[@key='EnableSymbolLoadingAtServerStartup']").Value = "$($enableSymbolLoadingAtServerStartup -eq $true)"
```

Comment:
@freddydk One question - do you think I could add another variable called for example `$enableSymbolLoadingAtServerStartupValid = $true` (assign it here in this block as $true)?

I think it could be useful later in other parts, in the overrides generally to identify if the running NAV version allows this feature.

I know I can identify it using the major version of NAV but not every major version greater than 10 would be compatible. Of course, if we can suppose that the previous dev-previews are not relevant and won\`t be used anymore by anyone then it could be ok.